client/Lykke.Service.PaymentSystem.Client/PaymentSystemClient.cs

```diff
@@ -0,0 +1,127 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Threading;
+using System.Threading.Tasks;
+using Common.Log;
+using Lykke.Service.PaymentSystem.Client.AutorestClient;
+using Lykke.Service.PaymentSystem.Client.AutorestClient.Models;
+
+namespace Lykke.Service.PaymentSystem.Client
+{
+    /// <summary>
+    /// PaymentSystemClient
+    /// </summary>
+    public class PaymentSystemClient : IPaymentSystemClient, IDisposable
+    {
+        private PaymentSystemAPI _service;
+
+        /// <summary>
+        /// Constructor
+        /// </summary>
+        /// <param name="serviceUrl">Service URL</param>
+        public PaymentSystemClient(string serviceUrl)
+        {
+            if (string.IsNullOrWhiteSpace(serviceUrl))
+                throw new ArgumentException("Value cannot be null or whitespace.", nameof(serviceUrl));
+
+            _service = new PaymentSystemAPI(new Uri(serviceUrl));
```

Comment:
need to add constructor (extend PaymentSystemAPI partial class) to prevent memory leak, and use it `new PaymentSystemAPI(new Uri(serviceUrl), new HttpClient())`

example: https://github.com/LykkeCity/Lykke.Service.Metrics/blob/test/client/Lykke.Service.Metrics.Client/MetricsAPI.Extend.cs